MaaSSim/utils.py

```diff
@@ -194,13 +193,13 @@ def generate_demand(_inData, _params=None, avg_speed=False):
                                                  replace=True).index)  # sample destination nodes from a distribution
 
     requests['dist'] = requests.apply(lambda request: _inData.skim.loc[request.origin, request.destination], axis=1)
-    while len(requests[requests.dist >= _params.dist_threshold]) > 0:
+    while len(requests[requests.dist >= _params.dist_threshold_max]) + len(requests[requests.dist < _params.dist_threshold_min]) > 0:
```

Comment:
hm.... please avoid renaming old parameters, so keep the 'dist_threshold' and add 'dist_threshold_min' 
I know it is less logical, but otherwise it will mess everything who has 'dist_threshold' and doesn't 'dist_threshold_max' ....

Reply:
I see, will change that